api/utils.py

```diff
@@ -752,3 +753,45 @@ def get_protocol_scheme(url):
     protocol = parsed_endpoint.scheme
 
     return protocol
+
+
+def get_fairsharing_metadata(offline=True, username="", password="", path=""):
+    if offline == True:
+        f = open(path)
+        fairlist = json.load(f)
+        f.close()
+
+    else:
+        url = "https://api.fairsharing.org/users/sign_in"
+        payload = {"user": {"login": username, "password": password}}
+        headers = {"Accept": "application/json", "Content-Type": "application/json"}
+
+        response = requests.request(
+            "POST", url, headers=headers, data=json.dumps(payload)
+        )
+
+        # Get the JWT from the response.text to use in the next part.
+        data = response.json()
+        jwt = data["jwt"]
+
+        url = "https://api.fairsharing.org/search/fairsharing_records?page[size]=2500&fairsharing_registry=standard&user_defined_tags=metadata standardization"
+
+        headers = {
+            "Accept": "application/json",
+            "Content-Type": "application/json",
+            "Authorization": "Bearer {0}".format(jwt),
+        }
+
+        response = requests.request("POST", url, headers=headers)
+        fairlist = response.json()
+        user = open(path, "w")
+        json.dump(fairlist, user)
+        user.close()
+    return fairlist
+
+
+def check_fairsharing_abbreviation(fairlist, abreviation):
+    for standard in fairlist["data"]:
+        if abreviation == standard["attributes"]["abbreviation"]:
+            return (100, "Your metadata standard appears in Fairsharing")
+    return (0, "Your metadata standard has not been found in Fairsharing")
```

Comment:
this code should be moved to the RDA indicator function. Then, I would use a more concrete message for the specific indicator, e.g. "Metadata standard in use complies with a community standard according to FAIRsharing.org"

Reply:
Fixed in commit @PalomoIFCA
[Bug fix and simplified code by not using func](https://github.com/EOSC-synergy/FAIR_eva/pull/142/commits/166682559e247ce94a49679c5f3cd56da736b802)